lib/models/application.coffee

```diff
@@ -39,7 +39,9 @@ exports.getAll = (callback) ->
 		options:
 			orderby: 'app_name asc'
 			expand: 'device'
-	.then (applications) ->
+	.nodeify (error, applications) ->
```

Comment:
How does `applications` in `.map()` get dereferenced? It's outside of the scope in the place you're using it.